packages/next/cli/next-build.ts

```diff
@@ -54,6 +56,9 @@ const nextBuild: cliCommand = (argv) => {
   if (args['--no-lint']) {
     Log.warn('Linting is disabled')
   }
+  if (args['--no-mangling']) {
+    Log.warn('Mangling is disabled')
```

Comment:
Maybe this should provide a bit more information like that it's meant for debugging and affects performance?